### ISSUE TITLE
Please Read

### ISSUE THREAD

**Issue by meuserhere** [SubhamTyagi/loco-answers#268]
Accessibility doesn't work on HQ now. Please fix it soon because OCR mode gives too many wrong answers. Please fix
